activerecord/lib/active_record/persistence.rb

```diff
@@ -55,6 +57,201 @@ def create!(attributes = nil, &block)
         end
       end
 
+      # Inserts a single record into the databases. This method constructs a single SQL INSERT
+      # statement and sends it straight to the database. It does not instantiate the involved
+      # models and it does not trigger Active Record callbacks or validations. However, values
+      # passed to #insert will still go through Active Record's normal type casting and
+      # serialization.
+      #
+      # See <tt>ActiveRecord::Persistence#insert_all</tt> for documentation.
+      def insert(attributes, returning: nil, unique_by: nil)
+        insert_all([ attributes ], returning: returning, unique_by: unique_by)
```

Comment:
would be super-cool if create/save could *also* take a `returning` argument so that I could tell Rails the db is going to calculate some additional values it should set on the instance.  Does that seems like a reasons feature request? I could work on it.  I wouldn't normally bring it up this way, but it seems like this PR paves the way for something like this.